src/vm/core.zig

```diff
@@ -358,6 +358,56 @@ pub const CairoVM = struct {
         }
     }
 
+    /// Attempts to deduce `op1` and `res` for an instruction, given `dst` and `op0`.
+    ///
+    /// # Arguments
+    ///
+    /// - `inst`: The instruction to deduce `op1` and `res` for.
+    ///
+    /// - `dst`: The destination of the instruction.
+    ///
+    /// - `op0`: The first operand of the instruction.
+    ///
+    /// # Returns
+    ///
+    /// - `Tuple`: A tuple containing the deduced `op1` and `res`.
+    fn deduce_op1(self: *const CairoVM, inst: *const instructions.Instruction, dst: ?*const relocatable.MaybeRelocatable, op0: ?*const relocatable.MaybeRelocatable) !std.meta.Tuple(&[_]type{ ?relocatable.MaybeRelocatable, ?relocatable.MaybeRelocatable }) {
```

Comment:
Would be good to just add a comma at the end of this list for fmt :)

Reply:
Yep, just realized zig formatted it like this. 250 columns ez